runtime/src/main/resources/application.yml

```diff
@@ -80,3 +80,11 @@ generator:
 controllers:
   integration:
     enabled: true
+
+client:
+  state:
+    authenticationAlgorithm: HmacSHA1
```

Comment:
I'm don't fully understand, you mean have them as default so that they don't need to be specified or remove them entirely?